walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,8 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
+    name = name.trim();
+    name = name.replaceAll("[^A-Za-z0-9]","");
```

Comment:
Nit: This does not support my name if I type it with the accented Hungarian letters (`Tamás`) :P

Reply:
Now it should :D